pysindy/pysindy.py

```diff
@@ -362,6 +362,16 @@ def print(self, lhs=None, precision=3):
 
         precision: int, optional (default 3)
             Precision to be used when printing out model coefficients.
+
+        **kwargs: Additional keyword arguments passed to the print function:
+            - sep: str, optional (default=' ')
+                string inserted between values, default a space.
+            - end: str, optional (default='\\n')
+                string appended after the last value, default a newline.
+            - file: str, optional (default = None)
+                a file-like object (stream); defaults to the current sys.stdout.
+            - flush: bool, optional (default = False)
+                whether to forcibly flush the stream.
```

Comment:
```suggestion
        **kwargs: Additional keyword arguments passed to the builtin print function
```

The point of this is to allow SINDy.print to automatically allow all kwargs that builtin print allows **or will allow in future python versions**.  They don't need to be individually documented here, as a user can just check the builtin print documentation.